internal/configs/virtualserver.go

```diff
@@ -343,17 +354,42 @@ func generateUpstream(upstreamName string, upstream conf_v1alpha1.Upstream, isEx
 			MaxConns:    generateIntFromPointer(upstream.MaxConns, cfgParams.MaxConns),
 			Resolve:     isExternalNameSvc,
 		}
+
+		if isPlus {
+			s.SlowStart = generateSlowStartForPlus(upstream, lbMethod)
+		}
+
 		upsServers = append(upsServers, s)
 	}
 
 	return version2.Upstream{
 		Name:      upstreamName,
 		Servers:   upsServers,
-		LBMethod:  generateLBMethod(upstream.LBMethod, cfgParams.LBMethod),
+		LBMethod:  lbMethod,
 		Keepalive: generateIntFromPointer(upstream.Keepalive, cfgParams.Keepalive),
 	}
 }
 
+func generateSlowStartForPlus(upstream conf_v1alpha1.Upstream, lbMethod string) string {
+	slowStart := upstream.SlowStart
```

Comment:
we don't usually create auxiliary variables like this one. Note that is only used twice. Can we just use `upstream.SlowStart` instead?